src/acquisition/fluview/impute_missing_values.py

```diff
@@ -293,7 +296,15 @@ def impute_missing_values(database, test_mode=False):
             known_values["pr"] = (0, 0, 0)
 
         # get the imputation matrix and lists of known and unknown locations
-        F, known, unknown = get_fusion_parameters(known_values.keys())
+        try:
+            F, known, unknown = get_fusion_parameters(known_values.keys())
+        except StatespaceException as e:
+            message = str(e)
+            if message == UNDERDETERMINED_MSG:
+                warn(message)
+            else:
+                print(message)
+            continue
```

Comment:
`warn()` goes to stderr instead of stdout, and we can lose the log message.  might as well just print them all:
```suggestion
            # imputation cannot be performed for this issue/epiweek, log and move on
            print(e)
            continue
```